### ISSUE TITLE
Investigate `pflag.Set` being called many times on startup

### ISSUE THREAD

**Issue by shaupt131** [storj/storj#4433]
If you go into ```satellite/repair/checker/config.go``` and remove ```ros.List = nil``` from

```
// Set sets the value from a string in the format "k/o/n-override,k/o/n-override,...".
func (ros *RepairOverrides) Set(s string) error {
	ros.List = nil
```
then regenerate the satellite config lock file, you will see the generated config includes duplicates because `Set()` was called multiple times. Why is this happening and is there a way to prevent it? Low priority.

[Formerly DUR-63](https://storjlabs.atlassian.net/browse/DUR-63)

**Comment by iglesiasbrandon** [comment]
Hola @mobyvb do you know if this ticket is still valid? We are not sure what it is referring to.

**Comment by mobyvb** [comment]
@iglesiasbrandon yeah, I actually do remember this. I just tested it, and it is still something that happens. That said, it's probably super low priority. Here are some more specific reproduction steps if someone wants to take a look:
1. Update dev default for `RepairOverrides` in `satellite/repair/checker/config.go`, like this `devDefault:"2/4/8-3"`
2. delete or comment the `ros.List = nil` line mentioned in the body of this issue
3. `go install ./...`
4. `storj-sim network destroy`
5. `storj-sim network setup`
6. open satellite config, note `checker.repair-overrides: 2/4/8-3,2/4/8-3`

If you put the `ros.List = nil` line back in, and do steps 3-6 again, the config will have the override only once, as expected

**Comment by egonelbre** [comment]
You can print out the callstack on each call to `Set`, that should give information on why it's being called multiple times.

**Comment by iglesiasbrandon** [comment]
this issue is stale, if this is still a problem we will open a new issue.
